src/main/java/istjbot/command/DateCommand.java

```diff
@@ -0,0 +1,34 @@
+package istjbot.command;
+
+import istjbot.exception.BotException;
+import istjbot.storage.Storage;
+import istjbot.task.TaskList;
+import istjbot.ui.Ui;
+
+import java.time.format.DateTimeParseException;
+
+public class DateCommand extends Command {
```

Comment:
Perhaps add a header comment for this class?

I noticed this issue in some of the other command files too.